### PR TITLE
Add dates and improve note

### PR DESCRIPTION
**Pull request by jwfuller**
The Privacy SIG suggested two more additions

1. Last Updated and Last Reviewed dates. We want to encourage regular review of the form by module developers and POs. However, the data in the form may not change for years at a time, especially for very stable modules. We want to be able to demonstrate that even if the form hasn't been updated in 5 years, it may still be accurate.
2. Expanded note to make it clear that it is okay to add personal data items to the form as needed.

### CODE REVIEW

**Comment by inkuss** [comment]
reviewed, +1

File: PERSONAL_DATA_DISCLOSURE.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+_Last Updated: (date form was last changed)_
```
Comment (julianladisch) [inline PERSONAL_DATA_DISCLOSURE.md:1]: "Last Updated" duplicates the git commit history and therefore is not needed.

File: PERSONAL_DATA_DISCLOSURE.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+_Last Reviewed: (date form was last reviewed for accuracy)_
```
Comment (julianladisch) [inline PERSONAL_DATA_DISCLOSURE.md:2]: FOLIO uses Jira to track progress on tasks like this. Example, note "Issues in epic": https://issues.folio.org/browse/FOLREL-533

What advantage does this git based progress tracking have? Isn't it duplicating the "Status" field of the Jira issue?

Comment (dltj) [reply PERSONAL_DATA_DISCLOSURE.md:2]: As I recall, the reason for adding these lines to the PDD file was for automated reporting. We can write a script that scans the PDD files in FOLIO repositories, then uses the [Team vs module responsibility matrix](https://wiki.folio.org/display/REL/Team+vs+module+responsibility+matrix) to send follow-up messages when a module's PDD hasn't been reviewed in a while.  Modification date we could get from Git; review date would be harder.

File: PERSONAL_DATA_DISCLOSURE.md (hunk 1 above)
Comment (inkuss) [reply PERSONAL_DATA_DISCLOSURE.md:1]: +1

**Comment by inkuss** [comment]
very good

**Review by inkuss — APPROVED** [review]
kommentar

**Comment by inkuss** [comment]
Alles klar ! Mit Adam Chandler abgesprochen.

**Pull request merged**